TLM/TLM/U/UIScaler.cs

```diff
@@ -8,51 +8,28 @@ namespace TrafficManager.U {
     /// https://github.com/kianzarrin/Skylines-ModTools/blob/master/Debugger/UI/UIScaler.cs
     /// </summary>
     public static class UIScaler {
-        public static bool TryGetScreenResolution(out Vector2 resolution) {
-            UIView uIView = UIView.GetAView();
-            if (uIView) {
-                resolution = uIView.GetScreenResolution();
-                return true;
-            }
-
-            resolution = default;
-            return false;
-        }
-
-        private static float BaseResolutionX {
-            get {
-                if (TryGetScreenResolution(out Vector2 resolution)) {
-                    // 1920f if aspect ratio is 16:9;
-                    return resolution.x;
-                }
-
-                return 1080f * AspectRatio;
-            }
-        }
-
-        private static float BaseResolutionY {
-            get {
-                if (TryGetScreenResolution(out Vector2 resolution)) {
-                    // always 1080f. But we keep this code for the sake of future proofing
-                    return resolution.y;
-                }
+        private static float BaseResolutionX => UIView.GetAView().GetScreenResolution().x;
```

Comment:
I'd reduce GetAView() calls to a minimum.
Call it once and cache it.
The method searches every time for the first UIView even though the value never changes.
Also the method is implemented via LINQ and will cause heap allocations each time.